apps/theming-designer/src/shared/Typography.tsx

```diff
@@ -0,0 +1,14 @@
+import { ITextProps, Text } from 'office-ui-fabric-react';
+import * as React from 'react';
+
+const titleTextStyles: ITextProps['styles'] = (p, theme) => ({
+  root: {
+    display: 'block',
+    fontWeight: 600,
+    color: theme.palette.neutralPrimary,
+    fontSize: 20,
+    marginBottom: 20
+  }
+});
+
+export const TitleText = (p: ITextProps) => <Text styles={titleTextStyles} {...p} />;
```

Comment:
Couple things to consider just for conversation but not a change recommendation.

You added a reference to a palette color. Was there no semantic color? There is a high risk of a contrast problem here whenever we go directly to palette.

Hardcoding font propertiesmeans they are not theme-able. For an app that's not ideal, where the design language should really be centralized in the theme. What would you have preferred here?

```
fontSize: theme.fonts.sizes.s1;
```

or a loose variable with some assumption (fontSize pulls from theme.fonts.size)

```
fontSize: 's1'
```

You used `Text` but it provided no value for you at all. If you could define the variant in the theme, would that have helped?

```
theme = {
  fonts: {
    variants: {
       title: {
         size: 20,
         weight: 600
       }
     }
  }
};

<Text variant="title" color="semanticTextColor">`
```

Reply:
100% on all of those. i should be better pulling from the theme values. The challenge is translating from the Figma file to the proper values in the theme. Making it easier to find the theme variable that matches the value you are trying to apply would be great. i.e. what text colors use neutralPrimary (as the hex value in the figma file was the same as neutralPrimary)